doc/source/skips/1-governance.rst

```diff
@@ -145,16 +146,18 @@ are made according to the following rules:
   sentence (but no change of the scikit-image.org landing page or the “about”
   page), require approval by a core developer *and* no disagreement or requested
   changes by a core developer on the issue or pull request page (lazy
-  consensus). Core developers are expected to give “reasonable time” to others
-  to give their opinion on the pull request if they’re not confident others
-  would agree.
+  consensus). Core developers are expected to wait one or two days for others to give
+  their opinion if they’re not confident that others would agree.
 
-- **Code changes and major documentation changes** require agreement by *two*
-  core developers *and* no disagreement or requested changes by a core developer
-  on the issue or pull-request page (lazy consensus).
+- **Code and major documentation changes, and changes to the API** require agreement by
+  *two* core developers *and* no disagreement or requested changes by a core developer
+  on the issue or pull request page (lazy consensus). In the case of disagreement or
+  requested changes, core developers are expected to wait about a week for others to
+  give their opinion if they’re not confident that others would agree.
 
 - **Changes to the API principles** require a :ref:`SKIP <skip>` and follow the
-  decision-making process outlined above.
+  decision-making process outlined above. But, in this case, the objection period
+  should be one-month-long.
```

Comment:
Ok, I'm fine with either time periods; @stefanv we can merge the PR with this change:
```suggestion
  should be two-week-long.
```

Reply:
That's a pretty substantial changes given that other's have already approved this. So I'd rather merge this as is for now and see what kind of experience we have with this. I imagine one-month is totally fine since I would start counting once the SPEC is going live / leaves draft mode.

Reply:
OK, merging this for now; btw it should probably be "the objection period should be one month" or "one month long".

Reply:
Actually, going to let you fix that and then merge :)

Reply:
> That's a pretty substantial change given that other's have already approved this. So I'd rather merge this as is for now and see what kind of experience we have with this. I imagine one-month is totally fine since I would start counting once the SPEC is going live / leaves draft mode.

Ok.

Reply:
> btw it should probably be "the objection period should be one month" or "one month long".

:thinking: I guess either work although they're different:

* "the period is one month" is definitely valid;
* "one-month-long" has adjective value as in "a one-month-long period" so you can say "the period is one-month-long" (right?).

Reply:
I believe you can use "a one-month-long period" or "the period is one month long", but not "the period is one-month-long".